pkg/core/manifest.go

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2018 The original author or authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package core
+
+import (
+	"fmt"
+	"gopkg.in/yaml.v2"
+	"io/ioutil"
+)
+
+const MANIFEST_VERSION = "0.1"
+
+// Manifest defines the location of YAML files for system components.
+type Manifest struct {
+	Version         string
+	IstioCRDs       string `yaml:"istio_crds"`
```

Comment:
The convention in yaml is camelCase, but I like [Thomas' suggestion](
https://github.com/projectriff/riff/pull/691/files#r212765973) better.

Reply:
Thanks. I'll fix this for now regardless of whether we adopt the grouping suggestion.